PowerValidatedSolutions.psm1

```diff
@@ -12841,6 +12841,106 @@ Function Import-vROPSNotification {
 }
 Export-ModuleMember -Function Import-vROPSNotification
 
+Function Test-vROPsAdapterStatus {
+    <#
+        .SYNOPSIS
+        Validates the integration status of a vROPs adapter through adapter's ID
+        
+        .DESCRIPTION
+        The Test-vROPsAdapterStatus cmdlet validates the integration status between vRealize Operations Manager and configured adapter.
+    
+        .EXAMPLE
+        PS C:\> Test-vROPsAdapterStatus -resourcelId "b214fd75-07cc-4dab-9fbb-95a6af739a04"
+        This example validate the integration status between vRealize Operations Manager and configured adapter through its ID. 
+    #>
+    
+    Param (
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$resourcelId
+    )
+    $name = (Get-vROPSAdapter | Where-Object { $_.id -eq $resourcelId }).resourceKey.name
+    write-output "Adapter Name : $($name)"
```

Comment:
We try not to have multiple Write-Output within a cmdlet. Can I suggest you add the Adapter Name as part of the Write-Output and Write-Error below?